apps/cow-tools/src/app/milkman/(components)/OrdersTable.tsx

```diff
@@ -109,14 +136,75 @@ function CancelButton({ status }: { status: string }) {
   );
 }
 
+function TransactionInfo({ order }: { order: AllSwapsQuery["swaps"][0] }) {
+  const priceCheckerInfo = getPriceCheckerInfoFromAddressAndChain(
+    order.chainId as 5,
+    order.priceChecker as Address,
+  );
+
+  const expecetedArguments = priceCheckerInfo?.arguments;
+
+  const decodedArgs = priceCheckerInfo
+    ? decodePriceCheckerData(
+        priceCheckerInfo,
+        order.priceCheckerData as `0x${string}`,
```

Comment:
```suggestion
        order.priceCheckerData as Address,
```

Reply:
Even that for us is the same thing this `priceCheckerData` is in bytes format, not an Address. I think that is more readable to let it is. wdyt?